src/zeit/content/volume/volume.py

```diff
@@ -63,6 +63,31 @@ def product(self, value):
     def covers(self):
         return zeit.content.volume.interfaces.IVolumeCovers(self)
 
+    @property
+    def previous(self):
+        return self._find_in_order(None, self.date_digital_published, 'desc')
+
+    @property
+    def next(self):
+        return self._find_in_order(self.date_digital_published, None, 'asc')
```

Comment:
Hast Du ne Idee? In zeit.web haben wir für nen ähnlichen Zweck `predecessor` und `successor`, das haut mich halt auch nicht so wirklich vom Hocker.